### ISSUE TITLE
Please add INR(₹) to supported currency list

### ISSUE THREAD

**Issue by RohanGupta15** [jerameel/sushi#15]
### Thank You for the app! 
I really liked your app with its minimal looks and it fits my needs perfectly.
However Indian Rupee is not in the supported currency list which makes it a bit difficult to use it as my main money manager. So it would be great if you can add that. 😃

**Comment by bagajohny** [comment]
Please dev add ₹

**Comment by jerameel** [comment]
Hey guys, this project uses numbro internally...

If you can add the currency on numbro that would be better.
https://github.com/BenjaminVanRyseghem/numbro/tree/develop/languages

but i can also add the language config within the project if you send me config. here's a reference for the config json.

```
{
  languageTag: 'fil-PH',
  delimiters: {
    thousands: ',',
    decimal: '.',
  },
  abbreviations: {
    thousand: 'k',
    million: 'm',
    billion: 'b',
    trillion: 't',
  },
  ordinal: (number) => {
    let b = number % 10;
    // eslint-disable-next-line no-bitwise
    return ~~((number % 100) / 10) === 1
      ? 'th'
      : b === 1
      ? 'st'
      : b === 2
      ? 'nd'
      : b === 3
      ? 'rd'
      : 'th';
  },
  currency: {
    position: 'prefix',
    symbol: '₱',
    code: 'PHP',
  },
  currencyFormat: {
    thousandSeparated: true,
    totalLength: 4,
    spaceSeparated: true,
    spaceSeparatedCurrency: true,
    average: true,
  },
  formats: {
    fourDigits: {
      totalLength: 4,
      spaceSeparated: true,
      average: true,
    },
    fullWithTwoDecimals: {
      thousandSeparated: true,
      mantissa: 2,
    },
    fullWithTwoDecimalsNoCurrency: {
      mantissa: 2,
      thousandSeparated: true,
    },
    fullWithNoDecimals: {
      output: 'currency',
      thousandSeparated: true,
      mantissa: 0,
    },
  },
}
```

**Comment by jerryn70** [comment]
+1 👍 for indian currency

**Comment by shambu2k** [comment]
There seems to be a pull request with config in numbro for the Indian rupee. https://github.com/BenjaminVanRyseghem/numbro/pull/650

**Comment by jerameel** [comment]
Looks like it would be merged to Numbro soon, then I can just simply update the numbro package. 

if it takes a week or so for it to get merged, I'll manually copy the currency data so we can have it on our end.

**Comment by shambu2k** [comment]
> Looks like it would be merged to Numbro soon, then I can just simply update the numbro package. 
> 
> if it takes a week or so for it to get merged, I'll manually copy the currency data so we can have it on our end. 

It would be better to wait for numbro to merge. There seems to be some internal changes to support the Indian Rupee number system.

**Comment by theY2Kbug** [comment]
Yes. I have made some internal changes... I feel it's better to wait for the merge of pull request.

**Comment by officialpiyush** [comment]
any updates on this?

**Comment by IsotoxalDev** [comment]
Can a temporary fix for this be done i really like this app so please add a temporary fix until the numbro has added the en-in

**Comment by SubhamV** [comment]
@jerameel Any update on this? Would really appreciate if this could be included in the app :)
Also, as per, [https://github.com/BenjaminVanRyseghem/numbro/pull/650](url), it appears that numbro has added INR (Indian Rupee).

**Comment by jerameel** [comment]
the PR is still open, that one you linked is the old PR @SubhamV 

This is the latest one
https://github.com/BenjaminVanRyseghem/numbro/pull/655

@theY2Kbug how can i use your branch instead? as it seems numbro is not actively maintained right now.

**Comment by jerryn70** [comment]
Any update on this ?

**Comment by theY2Kbug** [comment]
@jerameel @IsotoxalDev @jerryn70 

Let's wait. The maintainer is just being a giant dick about including "Indian names" in the "core of the library." Let's see what he says. If he seen-zones as usual, I'll pull this code and add it here but it needs to maintained actively.

**Comment by bagajohny** [comment]
> @jerameel @IsotoxalDev @jerryn70
> 
> Let's wait. The maintainer is just being a giant dick about including "Indian names" in the "core of the library." Let's see what he says. If he seen-zones as usual, I'll pull this code and add it here but it needs to maintained actively.

Thank you for your contribution. I saw his replies and it does seem like he is not at all interested. But I also do wonder how many of us would have personal finance numbers in "lakhs & crores" 😂

**Comment by theY2Kbug** [comment]
@bagajohny 

It doesn't matter how many of us use INR or the denomination style. Do you want it or not?

**Comment by bagajohny** [comment]
> @bagajohny
> 
> It doesn't matter how many of us use INR or the denomination style. Do you want it or not?

Yes, I want it. I was just saying if its too much hassle to add "lakhs & crores" then you can just keep the current mil & bil as it is. Just adding INR symbol will be enough. I was trying to make a joke about personal finance being in crores, I mean why would anyone who spends that much would be using this app?

**Comment by IsotoxalDev** [comment]
Any update on this?
